scripts/esbuild.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-console */
+const esbuild = require('esbuild');
+
+const production = process.argv.includes('--production');
+const watch = process.argv.includes('--watch');
+
+/**
+ * @type {import('esbuild').Plugin}
+ */
+const esbuildProblemMatcherPlugin = {
+  name: 'esbuild-problem-matcher',
+
+  setup(build) {
+    build.onStart(() => {
+      console.log('[watch] build started');
```

Comment:
`[watch]`?

Reply:
I think this is a bug where they should only include it when in watch mode. I updated.